README.md

```diff
@@ -1,7 +1,12 @@
 # IBM Dolomite Engine
 
-This repo was extracted from https://github.com/ibm-granite/dolomite-engine
-
-Original contributors:
+This repo contains code for IBM models, extracted from https://github.com/ibm-granite/dolomite-engine; all credit goes to the original authors:
 - mayank31398
 - kmehant 
```

Comment:
You can drop mehant, he is not an author

Reply:
done